### ISSUE TITLE
Icon problem in the title barBug title

### ISSUE THREAD

**Issue by tingjiangcao** [microsoft/WinUI-Gallery#1512]
### Which version of the app?

WinUI 3 Gallery

### Description

In "Windows + TAB" or "ALT + TAB" interface, the title bar icon shows the default icon instead of the app's icon.

### Screenshots

![Snipaste_2024-03-24_17-56-33](https://github.com/microsoft/WinUI-Gallery/assets/51285674/daddc129-4335-46ba-8e34-b87024c0734f)

Currently need "AppWindow.SetIcon("xxx")" to solve the problem

I think after setting the icon in "Package.appxmainfest", the titlebar icon should be set at the same time, instead of setting it manually. (SDK optimization suggestion)

**Comment by HO-COOH** [comment]
Looks like it's already fixed. There is a `appWindow.SetIcon("Assets/Tiles/GalleryIcon.ico");` in `NavigationRootPage.xaml.cs`

**Comment by Fa1ki** [comment]
WinUI 3 Gallery 2.4.3.0

There is also no icon in the task manager and in the hover window in the taskbar.

**Screenshots**
![Screenshot 2024-07-08 191743](https://github.com/microsoft/WinUI-Gallery/assets/112784508/4c861244-3975-4233-98f6-9bf1aa17ba42)
![Screenshot 2024-07-08 191924](https://github.com/microsoft/WinUI-Gallery/assets/112784508/fe2cdb35-22b7-48d6-ae72-0e4ae3dc5023)

**Comment by Jay-o-Way** [comment]
status = fixed

**Comment by HEIC-to-JPEG-Dev** [comment]
> status = fixed

And when will the fix be released - current release doesn't have the fix ?

**Comment by niels9001** [comment]
@marcelwgn this isn't fixed, right? due to some platform bug?

**Comment by ghost1372** [comment]
> [@marcelwgn](https://github.com/marcelwgn) this isn't fixed, right? due to some platform bug?

Store version is not fixed, but i am running Gallery with Visual Studio and issue is fixed

**Comment by marcelwgn** [comment]
Yeah there is some issue with the behavior changing when running packaged & published compared to running under VS

**Comment by duckheadsoftware** [comment]
So, still not fixed 2 years after reporting. Here's the 2026 v2.8.0.0 icons.

This is basics - The Windows Taskbar Icon !!

Taskbar
<img width="221" height="58" alt="Image" src="https://github.com/user-attachments/assets/e3f00af4-2187-4439-8434-7ab0feb68f69" />

Taskbar App Preview

<img width="282" height="181" alt="Image" src="https://github.com/user-attachments/assets/32e4df48-9577-45ec-a7be-f22814aa1f5a" />
